### PR TITLE
Disable access logging option

### PR DESCRIPTION
**Pull request by zimathias**

### CODE REVIEW

File: src/server/request_handler.cpp [hunk 1]
```diff
@@ -64,17 +64,19 @@ void RequestHandler::HandleRequest(const http::request &current_request, http::r
...
-            << (time_stamp->tm_mday < 10 ? "0" : "") << time_stamp->tm_mday << "-"
-            << (time_stamp->tm_mon + 1 < 10 ? "0" : "") << (time_stamp->tm_mon + 1) << "-"
-            << 1900 + time_stamp->tm_year << " " << (time_stamp->tm_hour < 10 ? "0" : "")
-            << time_stamp->tm_hour << ":" << (time_stamp->tm_min < 10 ? "0" : "")
-            << time_stamp->tm_min << ":" << (time_stamp->tm_sec < 10 ? "0" : "")
-            << time_stamp->tm_sec << " " << current_request.endpoint.to_string() << " "
-            << current_request.referrer << (0 == current_request.referrer.length() ? "- " : " ")
-            << current_request.agent << (0 == current_request.agent.length() ? "- " : " ")
-            << request_string;
+        if(!std::getenv("DISABLE_ACCESS_LOGGING")) {
```
Comment (TheMarex) [inline src/server/request_handler.cpp]: Please use our formatting style `.clang-format` in the project root.

**Comment by zimathias** [comment]
@TheMarex done!

**Comment by TheMarex** [comment]
@zimathias great. If you rebase on `master` this is good to merge.

**Comment by zimathias** [comment]
@TheMarex rebased on master.

**Pull request merged**